db/app/app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const { getTopics } = require("./controller");
+const { handlePsqlErrors, handleCustomErrors, handleServerErrors } = require("./errors");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
Please remove all unused code - this is not needed until handling request data from the client side.